cmd/kube-proxy/app/server.go

```diff
@@ -153,6 +153,7 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 
 	fs.BoolVar(&o.CleanupAndExit, "cleanup", o.CleanupAndExit, "If true cleanup iptables and ipvs rules and exit.")
 	fs.BoolVar(&o.CleanupIPVS, "cleanup-ipvs", o.CleanupIPVS, "If true and --cleanup is specified, kube-proxy will also flush IPVS rules, in addition to normal cleanup.")
+	fs.MarkDeprecated("cleanup-ipvs", "In a future release, running --cleanup will always flush IPVS rules")
```

Comment:
@gongguan can you open an issue to track this work please?

Reply:
Can [#78797](https://github.com/kubernetes/kubernetes/issues/78797) be used as the tracking issue or a new one raised by me?

Reply:
Should I add a comment with the tracking issue above the sentence?

Reply:
Ah, yes it can, just make sure the issue name is updated and it's not closed when this PR is merged

Reply:
rebased current code to pass the failed check. Could you please label an ok-to-test?